src/components/select/multi-select/multi-select.component.js

```diff
@@ -457,11 +456,12 @@ const MultiSelect = React.forwardRef(
     );
 
     return (
-      <StyledMultiSelect
+      <StyledSimpleSelect
```

Comment:
there was `position: relative` attribute on that element, are you sure it's not needed here anymore?

Reply:
I noticed that but was unable to find anything breaking after removal of the `position: relative`